README.md

```diff
@@ -78,6 +78,24 @@ curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/inst
 1. Build Protostar: `poe build`
     - You can find the newly created binary at `dist/protostar/protostar`
 
+#### Bumping cairo bindings version
+To bump cairo bindings version to the latest commit on fork (master branch), run: 
+
+```shell
+poetry run poe bump_cairo_bindings
+```
+
+After this, you can pull & rebuild your local version of bindings by running:
+```shell
+poetry run poe install_cairo_bindings
+```
+
+##### Caveats:
+Remember to have your working tree clean, since it creates a commit on the current branch.
```

Comment:
Maybe it shouldn't commit on the current branch then 🤔 Not sure, your call

Reply:
We should be ok. this is not a big deal since it's a development script.